lua/refactoring/debug/print_var.lua

```diff
@@ -0,0 +1,44 @@
+local Pipeline = require("refactoring.pipeline")
+local Point = require("refactoring.point")
+local Region = require("refactoring.region")
+local refactor_setup = require("refactoring.tasks.refactor_setup")
+local post_refactor = require("refactoring.tasks.post_refactor")
+local lsp_utils = require("refactoring.lsp_utils")
+local debug_utils = require("refactoring.debug.debug_utils")
+
+local function get_variable()
+    local variable_region = Region:from_current_selection()
+    return variable_region:get_text()[1]
+end
+
+local function printDebug(bufnr, config)
+    return Pipeline
+        :from_task(refactor_setup(bufnr, config))
+        :add_task(function(refactor)
+            local opts = refactor.config:get()
+            local point = Point:from_cursor()
+            local region = point:to_region()
+            -- always go below for text
+            opts.below = true
+            point.col = opts.below and 100000 or 1
+
+            -- Get variable text
+            local variable = get_variable()
```

Comment:
get text?

Perhaps we should separate node retrieval from text grabbing?

That way we could also pass in the same node to debug path serializer

Reply:
you can keep it